docs/website/docs/walkthroughs/configure-a-pipeline.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Configure a pipeline
+description: How to configure a pipeline
+keywords: [how to, configure a pipeline]
+---
+
+# Configure a pipeline
+
+To configure your pipeline in `dlt`, you can use providers to set up your configurations. These configurations can be specified using one or a combination of the following methods:
+
+1. **Environment Variables:** Set configuration parameters directly in your environment, which `dlt` can read at runtime.
+2. **Vaults:** Use secure vault services such as Airflow, Google Cloud, AWS, or Azure to manage and retrieve configuration parameters.
```

Comment:
imho this is very vague, we should have a link pointing on how to do this or some example code for the vaults.